hazelcast/src/main/java/com/hazelcast/spi/impl/proxyservice/impl/DistributedObjectEventPacket.java

```diff
@@ -68,12 +68,9 @@ public void readData(ObjectDataInput in) throws IOException {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder();
-        sb.append("DistributedObjectEvent");
-        sb.append("{eventType=").append(eventType);
-        sb.append(", serviceName='").append(serviceName).append('\'');
-        sb.append(", name=").append(name);
-        sb.append('}');
-        return sb.toString();
+        return "DistributedObjectEvent"
+                + "{eventType=" + eventType
+                + ", serviceName='" + serviceName + '\''
+                + ", name=" + name + '}';
```

Comment:
Opening brace should be on first line in `"DistributedObjectEvent"`.

Normally we have the ` + '}';` on a new line on multi-line messages.